src/status_im/ui/components/tabbar/styles.cljs

```diff
@@ -46,6 +46,17 @@
    :top      0
    :position :absolute})
 
+(def counter-public
+  {:right            3
+   :top              0
+   :position         :absolute
+   :background-color colors/blue
+   :border-radius    6
+   :width            12
+   :height           12
+   :border-width     2
+   :border-color     :white})
```

Comment:
To avoid background color leak around the border you can use 2 circles:
```suggestion
(def counter-public-container
  {:right            3
   :top              0
   :position         :absolute
   :border-radius    8
   :width            16
   :height           16
   :justify-content  :center
   :align-items      :center
   :background-color :white})

(def counter-public
  {:background-color colors/blue
   :width            12
   :border-radius    6
   :height           12})
```